### ISSUE TITLE
Error in Danish morningstar requests (different from time to time)

### ISSUE THREAD

**Issue by tobiasarleth** [joseballester/muFunds#58]
Hello José Ballester and muFunds

Thanks for a nice add-on (most of the time).

I am using muFunds to track three Danish investments through morningstar:
Sparindex INDEX DJSI World KL (ticker: SPIDKJWKL)
Sparindex INDEX  Europa Growth KL (ticker: SPIEUGKL)
Sparindex INDEX Emerging Markets KL (ticker: SPIEMIKL)

The weird thing is that somestimes one of them works and the two others don't. Then at other times, a different one works and the two others don't. So it differs from time to time which one or more that don't seem to work. If it makes an error, it goes:

Exception: Request failed for https://www.morningstar.dk returned code 403. Truncated server response: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd"> <HTML><HEAD><META HTTP-EQUIV="Content-Type" ... (use muteHttpExceptions option to examine full response) (linje 70).

For each ticker, I use this code:
SPIDJWKL: =E14*muFunds("nav";"DK0010297464";"morningstar-dk")
SPIEUGKL: =E15*muFunds("nav";"DK0010297548";"morningstar-dk")
SPIEMIKL: =E16*muFunds("nav";"DK0060300762";"morningstar-dk")

Sometimes it's my E14 that works fine and E15 and E16 gives me an error message and other days, E15 works fine and E14 and E16 gives me an error message without I touched anything. I don't understand this.

Can you help me?

Thanks.

**Comment by tobiasarleth** [comment]
Yes. Sounds as the same in #56 . No solution yet? It still happens from time to time, but I just checked today, and today all three codes in the above work fine. Halleluja.

**Comment by joseballester** [comment]
Closing this as it was an issue when pulling info from Morningstar country-specific pages, which are no longer used.
